circle.yml

```diff
@@ -92,6 +92,7 @@ jobs:
             docker-compose up -d
             sleep 20
             docker-compose ps
+            scripts/ui-test-hostname-setup.sh
```

Comment:
I tried to get login working with the old config but I needed to have everything route through localhost. This adds the IP of the addons-frontend image to the selenium-firefox image as ```localhost```.

Reply:
Maybe you can help me with that because I tried a bunch and I could never get it

Reply:
> Maybe you can help me with that because I tried a bunch and I could never get it

I'm merging this for now to unblock our CircleCI tests, let's follow-up this week to clean this up. I'll send you a mail.